skills_review/skills/management/commands/initialise_taxonomy.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand
+
+from skills_review.skills import taxonomy
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        taxonomy.create_job_title_embeddings()
+        print("Initialised")
```

Comment:
We could be more explicit here for what's been initialised - to futureproof against more commands being added without it being ambiguous with what's happened.

Reply:
Made it slightly more explicit